lang/stack/Portfile

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github                      1.0
+
+github.setup        commercialhaskell stack 2.1.1
+
+name                stack
+categories          lang haskell
+maintainers         nomaintainer
+license             BSD
+platforms           darwin
+
+description The Haskell Tool Stack
+long_description    ${description}. \
+    Stack is a cross-platform program for developing Haskell projects. \
+    It is intended for Haskellers both new and experienced.
```

Comment:
When you say "program", what do you mean? IDE? (Given that I can't tell, the description may need some work.)

Reply:
Copy-and-paste description text from their main page. I edited this a little.